spel/minimal-linux.json

```diff
@@ -212,7 +212,7 @@
             "name": "minimal-rhel-7.3-hvm",
             "type": "amazon-ebs",
             "ami_name": "{{ user `spel_identifier` }}-{{ build_name }}-{{ user `spel_version` }}.x86_64-gp2",
-            "ami_description": "This is an STIG-partitioned [**NOT HARDENED**], LVM-enabled, \"minimal\" RHEL 7.3 AMI, with \"billingProduct\" set for proper billing and licensed access to RedHat yum repos. This AMI contains updates through {{ isotime \"2006-01-02\" }}. For details, see {{ user `desc_url` }}.",
+            "ami_description": "This is a STIG-partitioned [**NOT HARDENED**], LVM-enabled, \"minimal\" RHEL 7.3 AMI (yum and license chargeback included) with updates through {{ isotime \"2006-01-02\" }}. See {{ user `desc_url` }} for details.",
```

Comment:
The description *can't* remain the same: CentOS doesn't have the license-related issues, so makes no sense to include verbiage around yum or chargeback.

While `billingProduct` may be a technically accurate topic of discussion, that's more for the "see details" link (or, if this were a MP item, the info/usage tab), its inclusion in the description blows out the character-limits. Eliminating it is what actually allowed the description to fit within 255 chars.